README.md

```diff
@@ -25,35 +25,32 @@ processors.
 
 ### Prerequisites
 
-- Python 3.6 or higher
+- Python `>=3.10`
 - PySCF library
 - Numpy
 - Scipy
-- libDMET (required for periodic BE)
-- [Wannier90](https://github.com/wannier-developers/wannier90)<sup>&&</sup> (to use Wannier functions)
-
-<sup>&&</sup>Wannier90 code is interfaced via [libDMET](https://github.com/gkclab/libdmet_preview) in QuEmb</sub>
-
-### Steps
-
-1. Clone the repository:
-   ```bash
-   git clone https://github.com/oimeitei/quemb.git
-   cd quemb
-
-2. Install QuEmb using one of the following approaches:
-    ```bash
-    pip install .
-    ```
-    or simply add `path/to/quemd` to `PYTHONPATH`
-    ```bash
-    export PYTHONPATH=/path/to/quemb:$PYTHONPATH
-    ```
-
-    For conda (or virtual environment) installations, after creating your environment, specify the path to mol-be source as a path file, as in:
-    ```bash
-    echo path/to/quemb > $(python -c "from distutils.sysconfig import get_python_lib; print(get_python_lib())")/quemb.pth
-    ```
+- [libDMET](https://github.com/gkclab/libdmet_preview) (required for periodic BE)
+- [Wannier90](https://github.com/wannier-developers/wannier90)<sup>##</sup> (to use Wannier functions)
+
+<sup>##</sup> `Wannier90` code is optional and only necessary to use Wannier functions in periodic code. </sub>
+
+The required dependencies, with the exception of the optional `Wannier90`,
```

Comment:
We also don't automatically install the SHCI from pyscf (https://github.com/pyscf/cornell-shci), right? Maybe we put off mentioning it alongside Wannier90 until we revisit and test SCI-BE

Reply:
I think the same is true for DMRG, isn't it @ShaunWeatherly ? 
I will add a footnote for both.

Reply:
after talking to @lweisburn I would postpone (#67 ) this to a point where we have working tests for both `SHCI` and `DMRG`. It sounds a bit dangerous to recommend optional packages to users otherwise.